hnn_core/tests/test_cell_response.py

```diff
@@ -44,6 +44,8 @@ def test_cell_response(tmpdir):
     empty_spike = CellResponse(spike_times=[[], []], spike_gids=[[], []],
                                spike_types=[[], []])
     empty_spike.write(tmpdir.join('empty_spk_%d.txt'))
+    empty_spike.write(tmpdir.join('empty_spk.txt'))
+    empty_spike.write(tmpdir.join('empty_spk_{0}.txt'))
```

Comment:
Does this overwrite the files? Then L49 is only testing the previous line

Reply:
yeah the others are smoke tests